instrumentation/grpc/Gemfile

```diff
@@ -9,9 +9,7 @@ source "https://rubygems.org"
 # Specify your gem's dependencies in opentelemetry-instrumentation-grpc.gemspec
 gemspec
 
-gem "rake", "~> 13.0"
-gem "rspec", "~> 3.0"
-gem "standard", "~> 1.3"
-gem "opentelemetry-sdk", "~> 1.0"
-gem "opentelemetry-test-helpers"
-gem "grpc"
```

Comment:
It looks like `rake` and `opentelemetry-test-helpers` aren't in the gemspec file. I think we may want to explicitly declare `rake`. If you're not using the `opentelemetry-test-helpers`, that one probably doesn't need to carry over.

Reply:
Thank you for spotting this. You're correct that `rake` should be included in the gemspec file. I've also added `opentelemetry-test-helpers`, as it's likely only a matter of time before it becomes necessary.

Reply:
Thank you!